### DISCUSSION TITLE
Possible to export to .dgn ?

### DISCUSSION THREAD

**Discussion by johnnyd710** [iTwin/itwinjs-core#2440] [Q&A] [answered]
I know how to export to IFC, but is it possible to export an imodel to ".dgn" ?

**Comment by jffmarker** [comment, marked as answer]
I think the closest you can get is to attach a reference to the iModel in the latest CONNECT Edition Power Products (like MicroStation), and grab representations of the elements through the reference to the master DGN. This is NOT meant to round-trip. I am not aware of anything just like the IFC exporter that we deploy. @abeesh @timlawrence-bentley

**Reply by candcconsulting** [reply]
You can also download a bim file and reference that into Microstation and Microstation based applications
